src/components/index/RequestsTableWithPagination.tsx

```diff
@@ -14,16 +14,23 @@ export const RequestsTableWithPagination = ({
   requests,
 }: Props) => {
   const elementCount = requests.length;
-  const paginateState = paginate({
+  const indices = page({
     currentPage,
     elementCount,
+    elementsPerPage: 25,
   });
 
   const paginatedRequests = requests.slice(
-    paginateState.startIndex,
-    paginateState.endIndex
+    indices.startIndex,
+    indices.endIndex
   );
 
+  const paginateState = paginate({
```

Comment:
This paginate state has the indices for slicing the rows, dont have to call 'page' separately